File: project-euler.java/src/main/java/euler/solutions/_1.java
package euler.solutions;

import euler.common.BaseSolution;

public class _1 extends BaseSolution {

    @Override
    public long solve() {
        int sum = 0;
        for (int i = 1; i < 1000; i++) {
            if (i % 3 == 0 || i % 5 == 0) {
                sum += i;
            }
        }
        return sum;
    }
}

File: project-euler.go/solutions/_1/1.go
package _1

func Solve() int64 {
	sum := 0
	for i := 1; i < 1000; i++ {
		if (i % 3 == 0 || i % 5 == 0) {
			sum += i;
		}
	}
	return int64(sum)
}
